examples/common/python/Makefile

```diff
@@ -15,14 +15,9 @@
 PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
 MOD_VERSION=${shell ../../../bin/get_version}
 
-# Format of wheel package is <pkg_name>-{python-tag}-{abi-tag}-{platform}
-WHEEL_FILE=dist/tcf_examples_common-${MOD_VERSION}-cp${PY_VERSION}-cp${PY_VERSION}m-linux_x86_64.whl
```

Comment:
Why is WHEEL_FILE variable removed ?

Reply:
wheel package naming convention is bit complex. Moreover, we really don't need suffix after the package name for install/uninstall using pip. One thing can be done is keeping the variable as below.
WHEEL_FILE=dist/tcf_example_common

Reply:
https://www.python.org/dev/peps/pep-0427/#file-name-convention. Let's follow existing whl file namng convention.  Format of wheel package is <pkg_name>-{python-tag}-{abi-tag}-{platform}

Reply:
ok